delphi/latents/cache.py

```diff
@@ -386,3 +395,11 @@ def save_config(self, save_dir: Path, cfg: CacheConfig, model_name: str):
                 config_dict = cfg.to_dict()
                 config_dict["model_name"] = model_name
                 json.dump(config_dict, f, indent=4)
+
+    def save_firing_counts(self):
+        """
+        Save the firing counts for the cached latents.
+        """
+        log_path = Path.cwd() / "results" / "log" / "hookpoint_firing_counts.pt"
```

Comment:
medium confidence: maybe we should pass the logging output path for the run into the latents cache init, and then use it here, and then in the CLI in \_\_main\_\_ we can include the name of the run in this path so the firing counts get saved to the particular run folder